aws_security_group.tf

```diff
@@ -112,296 +112,296 @@ resource "aws_security_group" "acme_web" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 23
     to_port     = 23
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   ingress {
     from_port   = 25
     to_port     = 25
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0","19.16.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
   }
 
   # HTTP access from the VPC
   ingress {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "192.164.0.0/24"]
   }
 
   # HTTPS access from the VPC
   ingress {
     from_port   = 443
     to_port     = 443
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "192.164.0.0/24"]
   }
 
   ingress {
     from_port   = 445
     to_port     = 445
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0", "192.164.0.0/24"]
+    cidr_blocks = ["<cidr>", "<cidr>"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: Unknown Port is exposed to the entire internet**
 
**Vulnerability**
Unknown Port is exposed to the entire internet
 
**Remediation**
Limit the access scope for a PORT to only allow access in internal networks and limited scope. If public interface exists, remove it and limit the access scope within the VPC only to applications or instances that requires access. Amazon Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html. As a further protection step, Use 'Tamper Protection' (Full Protection mode) and region lock to prevent unauthorized changes to network access settings, refer to /360003963194#Set_an_AWS_Security_Group_to_Full_Protection for more details.

source: accurics-